src/SW_Output.c

```diff
@@ -1091,36 +1050,36 @@ static void collect_sums(
 
 #ifdef STEPWAT
 static void _set_SXWrequests_helper(
+    SW_OUT_DOM *OutDom,

```

Comment:
The interface for `_set_SXWrequests_helper()` and `SW_OUT_set_SXWrequests()` can be simplified to no longer explicitly mention `timeSteps_SXW`.

Reply:
Good suggestion, let's do that!